.travis/publish_docs.sh

```diff
@@ -88,13 +102,21 @@ cp -r target/staging/* "$release_tag"/
 
 printf "${CYAN}Creating redirecting index.html.$RESET \n"
 mv target/build_index.sh "."
-sh ./build_index.sh "$release_tag"
+bash ./build_index.sh "$release_tag"
```

Comment:
I tried updating the shebang to use `#!/usr/bin/env sh` as suggested offline and tried running it. It gave me this error - `/home/travis/build/cerner/beadledom/.travis//publish_docs.sh: 22: /home/travis/build/cerner/beadledom/.travis//publish_docs.sh: Syntax error: "(" unexpected
`

Note: I tried on the different branch. Here is the diff - https://github.com/cerner/beadledom/compare/89d1955ea347...8251cf382e9b